### PR TITLE
Rewrite install on macOS article

### LINKED ISSUE #42365: Update mac install article for macOS 15.0 "Sequoia"
### Type of issue

Missing information

### Description

The rumor on news sites is that 15.0 is going to be released in Sept.

Once that happens, update this article with that information.

- Add .NET 8
- Reorganize the content
  The current structure is pretty sloppy. There is a lot of information in this article, it might be worth breaking it out into a few articles. Perhaps articles that are focused on troubleshooting, install, and general information.

Also, MacOS 12 ("Monterey") is out of support.

### Page URL

https://learn.microsoft.com/en-us/dotnet/core/install/macos#arm-based-macs

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/core/install/macos.md

### Document Version Independent Id

68d524f7-6cf1-2caa-56d5-f243952d1526

### Article author

@adegeo

### Metadata

* ID: 07a5fade-535f-2fad-c7b0-77bf9e28773e 
* Service: **dotnet-fundamentals**
* Sub-service: **install**

### PR DESCRIPTION
**Pull request by adegeo**
## Summary

- Rewrite doc for install-set-up-deploy template.
- Procedure steps now.
- New screenshots.
- Updated the steps in the validation includes.

Fixes #42365 

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/install/macos-notarization-issues.md](https://github.com/dotnet/docs/blob/6508643b88ce4fcae1e07836dc679f55ab26410c/docs/core/install/macos-notarization-issues.md) | [docs/core/install/macos-notarization-issues](https://review.learn.microsoft.com/en-us/dotnet/core/install/macos-notarization-issues?branch=pr-en-us-43045) |
| [docs/core/install/macos.md](https://github.com/dotnet/docs/blob/6508643b88ce4fcae1e07836dc679f55ab26410c/docs/core/install/macos.md) | [docs/core/install/macos](https://review.learn.microsoft.com/en-us/dotnet/core/install/macos?branch=pr-en-us-43045) |
| [docs/core/install/windows.md](https://github.com/dotnet/docs/blob/6508643b88ce4fcae1e07836dc679f55ab26410c/docs/core/install/windows.md) | [docs/core/install/windows](https://review.learn.microsoft.com/en-us/dotnet/core/install/windows?branch=pr-en-us-43045) |
| [docs/core/runtime-discovery/troubleshoot-app-launch.md](https://github.com/dotnet/docs/blob/6508643b88ce4fcae1e07836dc679f55ab26410c/docs/core/runtime-discovery/troubleshoot-app-launch.md) | [docs/core/runtime-discovery/troubleshoot-app-launch](https://review.learn.microsoft.com/en-us/dotnet/core/runtime-discovery/troubleshoot-app-launch?branch=pr-en-us-43045) |
| [docs/core/tools/dotnet-install-script.md](https://github.com/dotnet/docs/blob/6508643b88ce4fcae1e07836dc679f55ab26410c/docs/core/tools/dotnet-install-script.md) | [docs/core/tools/dotnet-install-script](https://review.learn.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script?branch=pr-en-us-43045) |

### CODE REVIEW

**Comment by adegeo** [comment]
@BillWagner

File: docs/core/install/macos.md [hunk 1]
```diff
@@ -15,134 +19,251 @@ ms.custom: linux-related-content
...
-[!INCLUDE [versions-not-supported](includes/versions-not-supported.md)]
+There are two runtimes you can install on macOS, and both are included with the SDK.
 
-## Runtime information
+- _ASP.NET Core Runtime_\
+  Runs ASP.NET Core apps. Includes the .NET runtime. **Not available as an installer.**
 
-The runtime is used to run apps created with .NET. When an app author publishes an app, they can include the runtime with their app. If they don't include the runtime, it's up to the user to install the runtime.
+- _.NET Runtime_\
+  This runs standard console-based .NET apps.
```
Comment (BillWagner) [inline docs/core/install/macos.md:51]: It might be worth saying that you can build ASP.NET web applications as a console app

**Review by BillWagner — APPROVED** [review]
This LGTM @adegeo 

I had one question, and then :shipit:

File: docs/core/install/macos.md (hunk 1 above)
Comment (adegeo) [reply docs/core/install/macos.md:51]:
```suggestion
  This runs normal .NET apps, but not specialized apps, such as apps built on ASP.NET Core.
```

**Pull request merged**